recipes/sphinxjp.themes.solarized/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "a9405e7fc875b9d5db80a8615ab7acb2cc6c98310ad65a1a7310652f0df8baaf" %}
+
+package:
+  name: sphinxjp.themes.solarized
+  version: {{ version }}
+
+source:
+  fn: sphinxjp.themes.solarized-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/s/sphinxjp.themes.solarized/sphinxjp.themes.solarized-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docutils
+    - sphinx
+    - sphinxjp.themecore
+
+  run:
+    - python
+    - docutils
+    - sphinx
+    - sphinxjp.themecore
+
+test:
+  imports:
+    - sphinxjp.themes
+    - sphinxjp
+    - sphinxjp.themes.solarized
+
+about:
+  home: http://bitbucket.org/miiton/sphinxjp.themes.solarized
+  license: MIT
```

Comment:
The license file was actually already in the tarball (in `src/LICENSE.txt`). Will fix this.